src/collar/oc_addons.lsl

```diff
@@ -436,7 +436,7 @@ state active
         }
 
 
-        if(iNum >= CMD_OWNER && iNum <= CMD_EVERYONE) UserCommand(iNum, sStr, kID);
+        if(iNum >= CMD_OWNER && iNum <= CMD_EVERYONE) UserCommand(iNum, sStr, kID);// Used to be CMD_WEARER.  What's the effect of the change?
```

Comment:
@Medea-Destiny I saw this changed in your branch from CMD_WEARER to CMD_EVERYONE.  I'd like to understand better what that does.

Reply:
In UserCommand that leads to the line:
` if(iNum>=CMD_OWNER && iNum<=CMD_WEARER) AddonsMenu(kID, iNum);`

So what happened is if you had a valid auth for the AddonsMenu it got checked twice, and if you didn't the collar was non-responsive to the click. 

In my edit we pass all valid users to the UserCommand and rely on filtering in the UserCommand to pass the AddonsMenu dialog to users with sufficient auth and present a no access error message to people who don't.

Reply:
Thank you!